core/frontend/src/tile/RealityModelTileTree.ts

```diff
@@ -452,6 +452,56 @@ export namespace RealityModelTileTree {
     const tileClient = new RealityModelTileClient(url, accessToken, iModel.contextId);
     const json = await tileClient.getRootDocument(url);
     let rootTransform = iModel.ecefLocation ? iModel.backgroundMapLocation.getMapEcefToDb(0) : Transform.createIdentity();
+    const geoConverter = iModel.noGcsDefined ? undefined : iModel.geoServices.getConverter("WGS84");
+    if (geoConverter !== undefined) {
+      let realityTileRange = RealityModelTileUtils.rangeFromBoundingVolume(json.root.boundingVolume)!.range;
+      if (json.root.transform) {
+        const realityToEcef = RealityModelTileUtils.transformFromJson(json.root.transform);
+        realityTileRange = realityToEcef.multiplyRange(realityTileRange);
+      }
+
+      const ecefOrigin = realityTileRange.localXYZToWorld(.5, .5, .5)!;

```

Comment:
@AlainRobertAtBentley  Can you take a look at the transform calculation here.   I'm not sure its not valid- but ecefEastPoint, ecefNorthPoint and particularly ecefElevationPoint are not east, north or elevation -- They are in ECEF coordinates here so adding Z is only going to be correct on the NorthPole.

Reply:
I would suggest renaming ecef<East, North> -> ecef<XPoint,YPoint>. I suppose East, North are still valid for GeoCoordinate values?